helmfile.d/stacks/falco.yaml

```
@@ -6,6 +6,17 @@ templates:
     labels:
       app: falco
 
+  falco-networkpolicy:
+    inherit:
+      - template: falco
+      - template: networkpolicies
+    installed: {{ and (.Values | get "falco.enabled" false) (.Values | get "networkPolicies.falco.enabled" false) }}
+    labels:
+      netpol: falco
```

Or does that get added some other way?
```suggestion
    labels:
      netpol: falco
      app: netpol
```

---

Yes it gets `policy: netpol` from [here](https://github.com/elastisys/compliantkubernetes-apps/blob/aarnq/falco-netpol-gen/helmfile.d/bases/environments.yaml#L43-L48) and `app: falco` from [here](https://github.com/elastisys/compliantkubernetes-apps/blob/aarnq/falco-netpol-gen/helmfile.d/stacks/falco.yaml#L3-L7). 

The `app: netpol` is deprecated in favour of `policy: netpol` to be able to include networkpolicies to specific things with the `app` labels.